docs/templates/plugin.rst.j2

```diff
@@ -1,6 +1,11 @@
 :source: @{ source }@
 
+{% if plugin_type == 'module' %}
 .. _@{ module }@:
+{% else %}
+.. _@{ plugin_type }@_@{ module }@:
```

Comment:
just use plugin type in all cases so then the ref can be explicit

Reply:
So all the anchors would change - the file module anchor would change from `.. _file` to `.. _module_file` and the file plugin anchor would change from `.. _file` to `.. _lookup_file`? I'll see how many references that would break.